src/components/classrooms/ClassroomEditor.jsx

```diff
@@ -177,9 +182,9 @@ const ClassroomEditor = (props) => {
             props.assignments[props.selectedClassroom.id] &&
             props.assignments[props.selectedClassroom.id].length > 0 &&
               students.map((student) => {
-                const galaxyAssignment = props.assignments[props.selectedClassroom.id].filter(
+                const galaxyAssignment = assignments.filter(
```

Comment:
[minor] If you're changing `props.assignments[props.selectedClassroom.id]` to the more sensible `assignments`, wouldn't you also want to update the condition checks as well? i.e.

```
{props.assignmentsStatus === ASSIGNMENTS_STATUS.SUCCESS &&
props.assignments[props.selectedClassroom.id] &&
props.assignments[props.selectedClassroom.id].length > 0 &&
  students.map((student) => {
...
```